spec/controllers/v1/events_controller_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe V1::EventsController, type: :controller do
+  describe "/v1" do
+    describe "GET #index" do
+      it_behaves_like "an index endpoint" do
+        let(:key) { :event_id }
+        # for underprivileged users, should only render events where the bag
+        # belongs to the user
+        let(:factory) do
```

Comment:
This is something I"d expect to see implemented as a fabricator.

Reply:
This is cribbed from https://github.com/mlibrary/chipmunk/blame/master/spec/controllers/v1/bags_controller_spec.rb#L11 - Or are you just saying that `Fabricate(:event, bag: Fabricate(:bag, user: user))` should get reduced down to something like `Fabricate(:event, bag_user: user: user)`?